.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -26,6 +26,14 @@ body:
   validations:
     required: true
 
+- type: input
+  attributes:
+    label: Helm chart version
+    description: Helm chart version that you encounterd problem.
```

Typo encounterd -> encountered. Maybe we better use a sentence like this:

```suggestion
    description: Version of the Helm chart this issue relates to
```

---

I fixed on https://github.com/argoproj/argo-helm/pull/1318/commits/160691bf74aeba807cf7a503726f531a5a99a98b 🙇🙇